src/api/faultmanagement/FaultmanagementAPI.js

```diff
@@ -0,0 +1,86 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2022, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+export default class FaultManagementAPI {
+    constructor(openmct) {
+        this.openmct = openmct;
+    }
+
+    acknowledgeFault(fault, ackData) {
+        return this.faultActionProvider.acknowledgeFault(fault, ackData);
+    }
+
+    addProvider(provider) {
+        this.provider = provider;
+    }
+
+    addFaultActionProvider(provider) {
+        this.faultActionProvider = provider;
+    }
+
+    getFaultModel(faultData, type) {
```

Comment:
The fault providers are responsible for returning faults in the correct model. We will need to document the model format so that developers can conform to it.

Reply:
Have removed this and added a JSDoc that can be referenced.